pkg/filter/filter.go

```diff
@@ -12,38 +14,51 @@ type Filter struct {
 	PartFilter PartFilter
 }
 
-func SetNDMFilters(vendorString, pathString, labelString string) []*Filter {
-	logrus.Info("register ndm filters")
-	listFilter := make([]*Filter, 0)
+func SetAutoProvisionFilters(devPathString string) []*Filter {
+	logrus.Info("register auto provision filters")
+
+	devPaths := strings.Split(devPathString, ",")
+	devPathFilter := RegisterDevicePathFilter(devPaths...)
+	return []*Filter{devPathFilter}
+}
+
+func SetExcludeFilters(vendorString, pathString, labelString string) []*Filter {
+	logrus.Info("register exclude filters")
+
+	vendors := strings.Split(vendorString, ",")
+	vendors = append(vendors, defaultExcludedVendors...)
+	vendorFilter := RegisterVendorFilter(vendors...)
+
+	paths := strings.Split(pathString, ",")
+	paths = append(paths, defaultExcludedPaths...)
+	pathFilter := RegisterPathFilter(paths...)
 
-	vendorFilter := RegisterVendorFilter(vendorString)
-	pathFilter := RegisterPathFilter(pathString)
-	labelFilter := RegisterLabelFilter(labelString)
-	listFilter = append(listFilter, vendorFilter, pathFilter, labelFilter)
+	labels := strings.Split(labelString, ",")
+	labelFilter := RegisterLabelFilter(labels...)
 
-	return listFilter
+	return append([]*Filter{}, vendorFilter, pathFilter, labelFilter)
```

Comment:
I think this could also be
```suggestion
	return []*Filter{vendorFilter, pathFilter, labelFilter)}
```

Reply:
Adrressed and rebased with ce09826.